src/EA.Weee.RequestHandlers.Tests.Unit/AatfReturn/NonObligated/AddNonObligatedHandlerTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.AatfReturn.NonObligated
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Domain.AatfReturn;
+    using EA.Weee.Domain.DataReturns;
+    using EA.Weee.Domain.Organisation;
+    using EA.Weee.RequestHandlers.AatfReturn;
+    using EA.Weee.RequestHandlers.Security;
+    using EA.Weee.Requests.AatfReturn.NonObligated;
+    using FakeItEasy;
+    using FakeItEasy.Configuration;
+    using FluentAssertions;
+    using RequestHandlers.AatfReturn.NonObligated;
+    using Xunit;
+
+    public class AddNonObligatedHandlerTests
+    {
+        private readonly IWeeeAuthorization authorization;
+        private readonly INonObligatedDataAccess nonObligatedDataAccess;
+        private readonly IReturnDataAccess returnDataAccess;
+        private readonly AddNonObligatedHandler handler;
+
+        private readonly Guid returnId = Guid.NewGuid();
+        private readonly Guid organisationId = Guid.NewGuid();
+        private readonly Return @return;
+
+        private AddNonObligated message;
+        private List<NonObligatedWeee> existingRecords;
+        private IEnumerable<NonObligatedWeee> submittedCollection;
+        private IReturnValueConfiguration<System.Threading.Tasks.Task<EA.Weee.Domain.AatfReturn.Return>> getReturnById;
+        private IReturnValueArgumentValidationConfiguration<System.Threading.Tasks.Task<System.Collections.Generic.List<EA.Weee.Domain.AatfReturn.NonObligatedWeee>>> fetchNonObligated;
+        private IReturnValueArgumentValidationConfiguration<System.Threading.Tasks.Task> sumbitCall;
+
+        public AddNonObligatedHandlerTests()
+        {
```

Comment:
not sure, but it feels like this should be in some sort of Before method, so these are reset each test